AMPDemos/index.html

```diff
@@ -227,15 +201,15 @@ <h1>AMP Demos</h1>
       </div>
     </div>
 
-    <!--     <div class="row">
-      <label class="label">Device</label>
+    <div class="row">
+      <label class="label">Location</label>
       <div class="entry">
-        <select name="device">
-          <option value="motoG4">Moto G4</option>
-          <option value="iPhone6">iPhone 6 iOS 12</option>
+        <select name="location" id="location-dropdown">
```

Comment:
can we set the default to MotoG4?

Reply:
where is the full locations list from? I don't see it hardcoded.

Reply:
@noodles

Reply:
2 defaults values are hardcoded in the HTML (Dulles & Sydney).
There is a javascript function that fetches the complete list from WPT & updates the contents of the dropdown.